app/src/main/res/values/strings.xml

```diff
@@ -664,4 +664,9 @@
     <string name="onboarding_theme_dark_title">Dark theme</string>
     <!-- Theme setting for light mode -->
     <string name="onboarding_theme_light_title">Light theme</string>
+
+    <!-- Text shown in snackbar when multiple tabs have been sent to device -->
+    <string name="sent_tabs_snackbar">Tabs sent!</string>
```

Comment:
We can use [plural string resources](https://developer.android.com/guide/topics/resources/string-resource#Plurals) for strings that change depending on the quantity.

Reply:
We can't :( Our l10n process can't deal with those resources yet

Reply:
Yeah, that's true. However we deferred implementing this until really really needed. If you want it then please let us know. :)

Reply:
@pocmo where I can create a feature request for that? 😬